### PR TITLE
Change autoware_auto_msgs to autoware_msgs

### PR DESCRIPTION
**Pull request by hsule**

### CODE REVIEW

File: zenoh_ros_type/autoware_auto_msgs/__init__.py [hunk 1]
```diff
@@ -1,2 +0,0 @@
-from .autoware_auto_control_msgs import *
```
Comment (evshary) [inline zenoh_ros_type/autoware_auto_msgs/__init__.py]: I think you can still reserve the old message type, but just use the new message type in the latest FMS.

File: zenoh_ros_type/autoware_auto_msgs/autoware_auto_vehicle_msgs.py [hunk 2]
```diff
@@ -1,75 +0,0 @@
-from dataclasses import dataclass
```
Comment (evshary) [inline zenoh_ros_type/autoware_auto_msgs/autoware_auto_vehicle_msgs.py]: Should we still need this?

**Review by evshary — CHANGES_REQUESTED** [review]

File: zenoh_ros_type/autoware_auto_msgs/__init__.py (hunk 1 above)
Comment (hsule) [reply zenoh_ros_type/autoware_auto_msgs/__init__.py]: These messages have been restored in the latest commit.

File: zenoh_ros_type/autoware_auto_msgs/autoware_auto_vehicle_msgs.py (hunk 2 above)
Comment (hsule) [reply zenoh_ros_type/autoware_auto_msgs/autoware_auto_vehicle_msgs.py]: autoware_auto_vehicle_msgs has been restored. However, FMS does not use these messages, so I didn't add these messages in the newly added autoware_msgs

**Review by evshary — APPROVED** [review]
LGTM

**Pull request merged**